IBF-Typhoon-model/setup.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Oct 31 16:01:00 2020
+
+@author: ATeklesadik
+"""
+from setuptools import setup
+PROJECT_NAME = "IBF-Typhoon-model"
+
+setup(
+    name=PROJECT_NAME,
+    version="0.1",
+    author="Aklilu Teklesadik",
+    author_email="ateklesadik@redcross.nl",
+    description="Typhoon impact forecasting model",
+    package_dir={'lib': ''},
```

Comment:
In the docker container, the package installation only works for me if this line is commented out